packages/azpipelines/BuildTasks/DeploySourceToOrgTask/task.json

```diff
@@ -9,7 +9,7 @@
     "version": {
         "Major": 9,
```

Comment:
Does this one need to be a major version change?

Reply:
Yes it needs to be UI change